src/Common/Types/QueryParamContextInterface.ts

```diff
@@ -117,4 +117,6 @@ export default interface QueryParamContextInterface {
   setPaymentRequest: Dispatch<SetStateAction<PaymentRequest | null>>;
   errors: SerializedError[] | null;
   setErrors: Dispatch<SetStateAction<SerializedError[] | null>>;
+  isApproved: boolean;
+  isTopProject: boolean;
```

Comment:
This should be a part of the project and not left in the wild. You should probably update the type `FetchedProjectDetails` instead.

Reply:
This was done, but `isApproved` and `isTopProject` were still present in the `QueryParamContext`. Removed in 5b723e77ab2c62355e5772afe38ad9f5379c2660